### PR TITLE
A0-3363: Runtime api for aleph key owner

### PR DESCRIPTION
**Pull request by ggawryal**
# Description

Add runtime api for getting owner of aleph key.

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)

# Checklist:

### CODE REVIEW

File: primitives/src/lib.rs [hunk 1]
```diff
@@ -283,6 +283,8 @@ sp_api::decl_runtime_apis! {
             session: SessionIndex
         ) -> Result<SessionCommittee<AccountId>, SessionValidatorError>;
         fn next_session_aura_authorities() -> Vec<AuraAuthorityId>;
+        /// Allows to get owner's AccountId for an Aleph key used in the current session.
+        fn aleph_key_owner(key: AuthorityId) -> Option<AccountId>;
```
Comment (deuszx) [inline primitives/src/lib.rs:287]: Is the `aleph_` prefix necessary? I understand that it's b/c this is from the "aleph sesion" but I haven't heard such concept before (maybe my lack of knowledge) and it seems that no other method in this API (`AlephSessionAPI`) is prefixed with it either.

**Review by deuszx — APPROVED** [review]
Feel free to ignore if I'm wrong.

**Review by kostekIV — APPROVED** [review]

File: primitives/src/lib.rs [hunk 2]
```diff
@@ -283,6 +283,8 @@ sp_api::decl_runtime_apis! {
             session: SessionIndex
         ) -> Result<SessionCommittee<AccountId>, SessionValidatorError>;
         fn next_session_aura_authorities() -> Vec<AuraAuthorityId>;
+        /// Allows to get owner's AccountId for an Aleph key used in the current session.
```
[the remark this answers is not in the document]
Comment (ggawryal) [reply primitives/src/lib.rs]: You're right, I was thinking from the `pallet_session`'s pov, in which this is called `aleph_key`, as opposed to `aura_key`. I agree that your suggestion is more clear generally.

File: primitives/src/lib.rs (hunk 1 above)
Comment (ggawryal) [reply primitives/src/lib.rs]: Yup, I wanted to distinguish `aleph_key` from `aura_key`, but this is rather unnecessary in `aleph` api.

**Pull request merged**